src/lib/EventListener/RequestAttributesListener.php

```diff
@@ -63,7 +63,7 @@ public function addRequestAttributes(FilterViewBuilderParametersEvent $event)
         $parameterBag = $event->getParameters();
 
         if ($parameterBag->has('locationId') && $request->get('_route') === '_ezpublishLocation') {
-            $location = $this->loadLocation($parameterBag->get('locationId'));
+            $location = $this->loadLocation($parameterBag->get('locationId'), $parameterBag->get('languageCode'));
```

Comment:
@andrerom Are you sure `languageCode` is always present in the ParameterBag? There is `hasContentLanguage` method call right after that block of code, so I'm not certain.

Reply:
Not 100% sure, just saw the code above and it will fallback to null if not set which fits.

Reply:
Checked now, yeah this is intentional. 
In loadLocation  I pass it along as `$languageCode ? [$languageCode] : null`